### PR TITLE
Added rndm wlcm msg func

### PR DESCRIPTION
**Pull request by muizzzafar11**
Added a list from which a random message will be sent if a user joins the server.

### CODE REVIEW

File: TU20Bot/Handler.cs [hunk 1]
```diff
@@ -19,6 +20,11 @@ public class Handler {
         private readonly CommandService commands;
         private readonly ServiceProvider services;
 
+        private string[] msgArr = { "Hello there!", "Whats poppin",
```
Comment (1whatleytay) [inline TU20Bot/Handler.cs]:
```suggestion
        private string[] greetings = {
            "Hello there!",
            "Whats poppin",
            "Wagwan",
            ...
```

 - I try not to use acronyms in my names too much.
 - Stuff like `arr` is describing the type, unlikely we need this in the name :D 
 - Give each entry on long lists their own line

File: TU20Bot/Handler.cs [hunk 2]
```diff
@@ -19,6 +20,11 @@ public class Handler {
...
+            "Wagwan", "Hi", "AHOY", "Welcome", "Greetings", "Howdy"};
+
+        private Random rndInt = new Random();
```
Comment (1whatleytay) [inline TU20Bot/Handler.cs]:
```suggestion
        private Random randomizer = new Random();
```

**Review by 1whatleytay — APPROVED** [review]
Nice work :D

File: TU20Bot/Handler.cs [hunk 3]
```diff
@@ -62,7 +68,7 @@ public class Handler {
         public async Task userJoined(SocketGuildUser user) {
             var channel = (IMessageChannel) client.GetChannel(737081385583378447);
```
Comment (mk5med) [inline TU20Bot/Handler.cs]: I would recommend placing the channel ID into a readonly variable so that we can easily identify what it is accessing.

**Review by 1whatleytay — APPROVED** [review]

**Pull request merged**